tests/e2e/cli/test_cli.py

```diff
@@ -125,26 +137,24 @@ def test_otx_e2e_cli(
         task_name in recipe
         for task_name in [
             "dino_v2",
+            "maskrcnn_r50_tv_tile",
         ]
     ):
         return
-    if task in ("visual_prompting", "zero_shot_visual_prompting"):
-        fxt_export_list = [
-            ExportCase2Test("ONNX", False, "exported_model_decoder.onnx"),
-            ExportCase2Test("OPENVINO", False, "exported_model_decoder.xml"),
-        ]  # TODO (sungchul): EXPORTABLE_CODE will be supported
-    elif "anomaly" in task:
-        fxt_export_list = [
-            ExportCase2Test("ONNX", False, "exported_model.onnx"),
-            ExportCase2Test("OPENVINO", False, "exported_model.xml"),
-        ]  # anomaly doesn't support exportable code
 
     overrides = fxt_cli_override_command_per_task[task]
     if "anomaly" in task:
         overrides = {}  # Overrides are not needed in export
 
     tmp_path_test = tmp_path / f"otx_test_{model_name}"
     for export_case in fxt_export_list:
+        if (
```

Comment:
How about using `ExportCase2Test` instead of apply this change? It seems that this change skip to check not only exportable code but also OV inference.